### PR TITLE
feat(clerk-js): Enforce aria-label in IconButton

### PR DESCRIPTION
**Pull request by desiprisg**
## Type of change

- [ ] 🐛 Bug fix
- [x] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [x] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [ ] `@clerk/types`
- [ ] `@clerk/themes`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend-core`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/edge`
- [ ] `build/tooling/chore`

## Description

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.

Before:
Screenshot 2022-12-02 at 00 35 33

After:
Screenshot 2022-12-02 at 00 34 14

### CODE REVIEW

File: packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationMembers.tsx [hunk 1]
```diff
@@ -53,8 +51,15 @@ export const OrganizationMembers = withCardStateProvider(() => {
           </Header.Root>
           {isAdmin && (
             <IconButton
+              aria-label='Invite'
```
Comment (nikosdouvlis) [inline packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationMembers.tsx:54]: Should this be based on the localisation key?

File: packages/clerk-js/src/ui/elements/IconButton.tsx [hunk 2]
```diff
@@ -1,21 +1,36 @@
...
 import { Button, Icon, useLocalizations } from '../customizables';
+import { ElementDescriptor } from '../customizables/elementDescriptors';
 import { PropsOfComponent } from '../styledSystem';
 
-export const IconButton = (props: PropsOfComponent<typeof Button> & { icon: React.ComponentType }) => {
-  const { children, icon, localizationKey, ...rest } = props;
+export const IconButton = (
+  props: PropsOfComponent<typeof Button> & {
+    icon: React.ReactElement | React.ComponentType;
+    'aria-label': string;
```
Comment (nikosdouvlis) [inline packages/clerk-js/src/ui/elements/IconButton.tsx:10]: Just to confirm, this already exists but we're making this required here right?

**Review by nikosdouvlis — APPROVED** [review]

File: packages/clerk-js/src/ui/elements/PasswordInput.tsx [hunk 3]
```diff
@@ -19,8 +20,10 @@ export const PasswordInput = (props: PropsOfComponent<typeof Input>) => {
         type={hidden ? 'password' : 'text'}
         sx={theme => ({ paddingRight: theme.space.$8 })}
       />
-      <Button
+      <IconButton
         elementDescriptor={descriptors.formFieldInputShowPasswordButton}
+        iconElementDescriptor={descriptors.formFieldInputShowPasswordIcon}
+        aria-label={`${hidden ? 'Show' : 'Hide'}password'`}
```
Comment (anagstef) [inline packages/clerk-js/src/ui/elements/PasswordInput.tsx:26]: ❓ Something is up with the quotes, right?

Comment (desiprisg) [reply packages/clerk-js/src/ui/elements/PasswordInput.tsx:26]: I did forget a space there :), but what do you mean?

File: packages/clerk-js/src/ui/elements/IconButton.tsx (hunk 2 above)
Comment (desiprisg) [reply packages/clerk-js/src/ui/elements/IconButton.tsx:10]: Not sure I understand. The aria label can be passed to all elements as far as I know, but it's optional. So yes, this is done to enforce the use of it in icon buttons.

File: packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationMembers.tsx (hunk 1 above)
Comment (desiprisg) [reply packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationMembers.tsx:54]: Yes probably. Should we tackle this in another PR along with any other similar strings that might be set?

File: packages/clerk-js/src/ui/elements/PasswordInput.tsx (hunk 3 above)
Comment (desiprisg) [reply packages/clerk-js/src/ui/elements/PasswordInput.tsx:26]: Ah yes, the last `'` shouldn't be there as well. Thanks. Doesn't cause any error so I missed it. My bad.

**Review by nikosdouvlis — APPROVED** [review]

**Pull request merged**